src/components/TitleCardList.tsx

```diff
@@ -152,52 +40,171 @@ const TitleCardList: React.FC<IProps> = ({
   icon,
   isCode,
 }) => (
-  <Table className={className}>
-    <TableHeader>
-      {icon && <StyledIcon name={icon} />}
+  <Box
+    bg="background"
+    boxShadow="tableBoxShadow"
+    width="100%"
+    margin="2rem"
+    border-radius="2px"
+    className={className}
+  >
+    <Flex
+      bg="ednBackground"
+      align="center"
+      justifyContent="space-between"
+      padding="1rem"
+      flexDirection="row-reverse"
+      fontWeight="600"
+      borderBottom="1px"
+      borderBottomStyle="solid"
+      borderBottomColor="text"
+    >
+      {icon && (
+        <ChakraIcon
+          as={Icon}
```

Comment:
Note: nothing to do from your side. This is a reminder for myself.

- [ ] @pettinarip to update this prop. We should not accept a string anymore, it should be the svg icon itself.

Context: we are deprecating the usage of our custom `Icon` component. https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md#update-dependencies